src/sonic-yang-mgmt/tests/libyang-python-tests/sample-yang-models/sonic-acl.yang

```diff
@@ -0,0 +1,274 @@
+module sonic-acl {
```

Comment:
Why do we have a sample for schema ?

In other words, why it is not checked in as proper schema ?

Reply:
- The yang models are in separate PR.   
- These sample models are used for testing the libyang  python APIs,   these APIs should work for any yang models. 
-  yang models may change/update, it's better to separate the yang models and the sample Yang models for testing libyang python APIs.